src/components/Cards/Status2.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import { styled } from '@mui/material/styles';
+import IconButton from '@mui/material/IconButton';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import Collapse from '@mui/material/Collapse';
+import ResourceDatagrid from '../Datagrid/ResourceDatagrid';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import SinglePieChart from '../Charts/SinglePieChart';
+const card = (
+    <React.Fragment>
+      <CardContent>
+        <Typography sx={{ fontSize: 14 }} color="text.secondary" gutterBottom>
```

Comment:
hardcoded value here, would be good if we can keep the classes and themes flexible for responsive design